src/routes/blog/ide-ux-bof-scipy-2024/+page.md

```diff
@@ -0,0 +1,141 @@
+The Nvidia folks mentioned that they do have a profiling tool that could potentially address some of those concerns, gives visibility on GPU utilization, bottlenecks and efficiency, and for interested folks to follow up with them directly.
+In response to people asking about support for such tools in a VSCode extension, they shared that these tools have their own UI but could explore IDE integration.
+They mentioned that tooling in Python tends to be "pretty thin" as users often don't go looking for tools and use the ones that are there, whereas their profiler can be really intimidating and throws a lot of  information at such users, potentially putting them off.
+
+All told, there was a surprising amount of interest and discussion around this once-niche topic, signaling that GPU compute has become something much closer to a mainstream need in the scientific space.
+Thus, while the discussion is of course of most interest to GPU companies like Nvidia, it suggests an IDE plugin to support this profiling tooling and other GPU integration features might be of significant benefit to the wider user community.
+
+
+## Spyder feedback
+
+While most of the discussion focused on topics applicable to broader categories of IDEs, there were a few specific bits of Spyder feedback.
+In particular, attendees praised Spyder's plotting functionality, though some expressed a preference for the non-default mode of displaying plots in separate windows over the Plots pane.
+Others mentioned that the one killer feature for them that it was missing was multi-line editing, a common request over the years and something we plan to work on for forthcoming Spyder versions, along with support for collaborative editing.
+
+A particularly salient comment was one attendees request that "We need RStudio for Python", with another replying that Spyder fills most of that gap.
+Perhaps the main RStudio feature not built in is Quarto/RMarkdown, though we are currently exploring the possibility of helping develop a plugin supporting that.
+Other comments, such as a desire for GitHub Copilot, better data exploration and visualization capabilities, and support for connecting to remote JupyterHub instances, are also features we already offer or are developing or exploring.
+
+
+## Next steps
+
+Overall, the broader learnings gained here for IDE and interface developers were perhaps less in the detailed specifics of each comment, but rather the common themes users focused the most on.
+Perhaps the least surprising was interest in generative AI, suggesting that IDE developers should continue to explore integration possibilities, but keeping a critical eye while doing so.
+By contrast, the most surprising was the extensive discussion about GPU compute and how IDEs and tools can help with that, which seems to indicate that IDE plugins for assisting such might be valuable.
```

Comment:
```suggestion
By contrast, the most surprising was the extensive discussion about GPU computing and how IDEs and tools can help with that, which seems to indicate that IDE plugins for assisting in this regard might be valuable.
```

- `compute` -> `computing` to go with my previous suggestions.
- It's easier for me (as a non-native speaker) to grasp the meaning of this sentence with the second suggestion.